fastlane/spec/actions_specs/spm_spec.rb

```diff
@@ -360,20 +360,7 @@
                 simulator: 'invalid_simulator'
               )
             end").runner.execute(:test)
-          end.to raise_error("Invalid simulator syntax. Please use 'iphonesimulator', 'ipadsimulator', or 'macossimulator'.")
-        end
-      end
-
-      context "when simulator is specified for non-package commands" do
-        it "raises an error" do
-          expect do
-            Fastlane::FastFile.new.parse("lane :test do
-              spm(
-                command: 'clean',
-                simulator: 'iphonesimulator'
-              )
-            end").runner.execute(:test)
-          end.to raise_error("Simulator is only applicable to package-related commands.")
+          end.to raise_error("Invalid simulator syntax. Please use 'iphonesimulator', or 'macossimulator'.")
```

Comment:
tests are failing, perhaps this will fix it:

```suggestion
          end.to raise_error("Please pass a valid simulator. Use one of the following: iphonesimulator, macossimulator"))
```

Reply:
Thanks for the help.